secboot/secboot_tpm.go

```diff
@@ -319,6 +319,7 @@ func SealKey(key EncryptionKey, params *SealKeyParams) error {
 	if err != nil {
 		return fmt.Errorf("cannot connect to TPM: %v", err)
 	}
+	defer tpm.Close()
```

Comment:
Nice catch! Would be great if we could also have a unit test for this but I looked and it seems like it's not super straightforward.